src/packet/opus.rs

```diff
@@ -23,8 +26,23 @@ impl Packetizer for OpusPacketizer {
     }
 
     fn is_marker(&mut self, data: &[u8], previous: Option<&[u8]>, last: bool) -> bool {
-        // TODO: dtx
-        false
+        // any non silenced packet would generally have more than 2 byts
```

Comment:
If there is one single marker packet (`self.marker = true`), then silence for 15 minutes, then again some audio (`is_marker = true`), this new audio packet would not go out as `marker` (bug)

Reply:
in case of no packets for a while there are two scenarios at play
* Packet loss: The speaker might still be speaking so can't determine for sure.
* Stopped speaker: In this case the packets would still be produced but at a lower frequency. This is also taken care of.